diff --git a/node_modules/@nuxt/ui/dist/runtime/components/Table.vue b/node_modules/@nuxt/ui/dist/runtime/components/Table.vue
index d11389f..0ee1aad 100644
--- a/node_modules/@nuxt/ui/dist/runtime/components/Table.vue
+++ b/node_modules/@nuxt/ui/dist/runtime/components/Table.vue
@@ -85,8 +85,12 @@ const paginationState = defineModel("pagination", { type: Object, ...{ default:
 const tableRef = ref(null);
 const tableApi = useVueTable({
   ...reactiveOmit(props, "as", "data", "columns", "caption", "sticky", "loading", "loadingColor", "loadingAnimation", "class", "ui"),
-  data,
-  columns: columns.value,
+  get data() {
+    return data.value
+  },
+  get columns() {
+    return columns.value
+  },
   meta: meta.value,
   getCoreRowModel: getCoreRowModel(),
   ...props.globalFilterOptions || {},
